projects/1.18/assets/many-ideas-doors/manyideas_doors/lang/zh_cn.json

```
@@ -0,0 +1,174 @@
+  "main_blocks_doors_special": "门：特殊",
+  "block.manyideas_doors.door_special_end": "末地门",
+  "block.manyideas_doors.door_special_police_box": "警察岗亭门",
+  "block.manyideas_doors.door_special_portcullis": "铁栅门",
+  "block.manyideas_doors.door_special_space": "太空门",
+  "main_blocks_doors_spruce": "门：云杉木",
+  "block.manyideas_doors.door_spruce_blank": "云杉木版门",
+  "block.manyideas_doors.door_spruce_bookshelf": "云杉木书架门",
+  "block.manyideas_doors.door_spruce_cassette": "云杉木格子门",
+  "block.manyideas_doors.door_spruce_french": "云杉木法式门",
+  "block.manyideas_doors.door_spruce_frosted": "云杉木磨砂门",
+  "block.manyideas_doors.door_spruce_glass": "云杉木玻璃门",
+  "block.manyideas_doors.door_spruce_heart": "云杉木心形装饰门",
+  "block.manyideas_doors.door_spruce_origin": "云杉木初始门",
+  "block.manyideas_doors.door_spruce_rustic": "云杉木乡村门",
+  "block.manyideas_doors.door_spruce_shoji": "云杉木障子门",
+  "main_blocks_doors_warped": "门：诡异木",
+  "block.manyideas_doors.door_warped_blank": "诡异木板门",
+  "block.manyideas_doors.door_warped_bookshelf": "诡异木书架门",
+  "block.manyideas_doors.door_warped_cassette": "诡异木格子门",
+  "block.manyideas_doors.door_warped_french": "诡异木法式门",
+  "block.manyideas_doors.door_warped_frosted": "诡异木磨砂门",
+  "block.manyideas_doors.door_warped_glass": "诡异木玻璃门",
+  "block.manyideas_doors.door_warped_heart": "诡异木心形装饰门",
+  "block.manyideas_doors.door_warped_origin": "诡异木初始门",
+  "block.manyideas_doors.door_warped_rustic": "诡异木乡村门",
+  "block.manyideas_doors.door_warped_shoji": "诡异木障子门",
+  "main_blocks_mini_lodges": "方块：迷你小屋",
+  "main_blocks_mini_lodges_outhouses": "迷你小屋：厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_acacia": "金合欢木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_birch": "白桦木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_crimson": "绯红木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_dark_oak": "深色橡木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_jungle": "丛林木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_oak": "橡木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_spruce": "云杉木厕所",
+  "block.manyideas_doors.mini_lodge_outhouse_warped": "诡异木厕所",
+  "block.manyideas_doors.mini_lodge_police_box": "警察岗亭",
+  "main_blocks_player_door_sensor": "方块：玩家门磁传感器",
+  "block.manyideas_doors.player_door_sensor": "玩家门磁传感器",
+  "main_messages": "信息",
+  "main_messages_redstone_key": "信息：红石钥匙",
+  "main_messages_redstone_key_sensor_range": "红石钥匙：传感器范围",
+  "main_messages_redstone_key_sensor_range_1": "传感器范围：1",
+  "message.manyideas_doors.1.description": "向下一个方块",
+  "message.manyideas_doors.1.title": "一",
+  "main_messages_redstone_key_sensor_range_2": "传感器范围：2",
+  "message.manyideas_doors.2.description": "向下两个方块",
+  "message.manyideas_doors.2.title": "二",
+  "main_messages_redstone_key_sensor_range_3": "传感器范围：3",
+  "message.manyideas_doors.3.description": "向下三个方块",
+  "message.manyideas_doors.3.title": "三",
+  "message.manyideas_doors.sensor_range": "传感器范围",
+  "main_item_groups": "物品组",
+  "itemGroup.manyideas_doors": "灵机妙想|门"
```

```suggestion
  "itemGroup.manyideas_doors": "灵机妙想丨门"
```
这个字符可以用拼音`gun`/`shu`打出来。
[sigh]